x-pack/plugins/case/server/client/cases/push.ts

```diff
@@ -0,0 +1,208 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import Boom from '@hapi/boom';
+
+import { flattenCaseSavedObject } from '../../routes/api/utils';
+
+import {
+  ActionConnector,
+  CaseResponseRt,
+  CaseResponse,
+  CaseStatuses,
+  ExternalServiceResponse,
+} from '../../../common/api';
+import { buildCaseUserActionItem } from '../../services/user_actions/helpers';
+
+import { CaseClientPush, CaseClientFactoryArguments } from '../types';
+import { createIncident, getCommentContextFromAttributes, isCommentAlertType } from './utils';
+
+export const push = ({
+  savedObjectsClient,
+  caseService,
+  caseConfigureService,
+  userActionService,
+  request,
+  response,
+}: CaseClientFactoryArguments) => async ({
+  actionsClient,
+  caseClient,
+  caseId,
+  connectorId,
+}: CaseClientPush): Promise<CaseResponse> => {
+  /* Start of push to external service */
+  const theCase = await caseClient.get({ id: caseId, includeComments: true });
+
+  if (theCase.status === CaseStatuses.closed) {
```

Comment:
let's put a mental note here that we will have to update this logic when we have sub case. Let be nice with future engineer ;)